src/rhsmlib/services/entitlement.py

```diff
@@ -69,18 +76,125 @@ def get_pools(self, pool_subsets=None, matches=None, pool_only=None, match_insta
         if 'consumed' in pool_subsets:
             results['consumed'] = self.get_consumed_product_pools(**options)
         if 'available' in pool_subsets:
-            results['available'] = self.get_available_pools(**options)
+            available = self.get_available_pools(**options)
+            results['available'] = [x._asdict() for x in available]
```

Comment:
Running following command: `$ busctl call com.redhat.RHSM1 /com/redhat/RHSM1/Entitlement com.redhat.RHSM1.Entitlement GetPools a{sv}a{sv} 1 pool_subsets s available 0` will be ended with following error:

```console
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/dbus/service.py", line 707, in _message_cb
    retval = candidate_method(self, *args, **keywords)
  File "<string>", line 2, in GetPools
  File "/home/jiri/github/candlepin/subscription_manager/src/rhsmlib/dbus/util.py", line 43, in dbus_handle_exceptions
    six.reraise(exceptions.RHSM1DBusException, "%s: %s" % (type(e).__name__, str(e)), trace)
  File "/home/jiri/github/candlepin/subscription_manager/src/rhsmlib/dbus/util.py", line 35, in dbus_handle_exceptions
    ret = func(*args, **kwargs)
  File "/home/jiri/github/candlepin/subscription_manager/src/rhsmlib/dbus/objects/entitlement.py", line 77, in GetPools
    pools = entitlement_service.get_pools(**options)
  File "/home/jiri/github/candlepin/subscription_manager/src/rhsmlib/services/entitlement.py", line 80, in get_pools
    results['available'] = [x._asdict() for x in available]
RHSM1DBusException: com.redhat.RHSM1.Error: AttributeError: 'dict' object has no attribute '_asdict'
```

Changing line 80 to this: `results['available'] = [x for x in available]` fixes the error, but I would prefer just use previous version: `results['available'] = self.get_available_pools(**options)`.